src/index.rs

```diff
@@ -120,7 +121,7 @@ impl IndexBuilder {
 
         let mut af = try!(AtomicFile::new(hunk_path));
         let compressed_len = report.measure_duration("index.compress",
-            || Snappy::compress_and_write(json_string.as_bytes(), &mut af))?;
+                                                     || Snappy::compress_and_write(json_string.as_bytes(), &mut af))?;
```

Comment:
(nit) I'm not sure what tool you're using but that doesn't look like an improvement.